myapp/app/models/task.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :bigint           not null, primary key
```

Comment:
we should use unsigned as we don't want negative id

i.e: `t.bigint :id, unsigned: true, null: false`

Reply:
cool I didn't know about unsigned. Learned~

Reply:
these notes are autogenerated by annotate gem, though it's not reflected we have changed the `int types` in these notes, it's been set [here](https://github.com/Fablic/training/pull/1461/files#r1046663783)